packages/nodes-base/nodes/Markdown/Markdown.node.ts

```diff
@@ -480,8 +480,7 @@ export class Markdown implements INodeType {
 						// eslint-disable-next-line n8n-nodes-base/node-param-default-wrong-for-simplify
 						default: false,
 						// eslint-disable-next-line n8n-nodes-base/node-param-description-wrong-for-simplify
-						description:
-							'Whether to parse line breaks as <br>, like GitHub does, without needing 2 spaces at the end of the line',
+						description: 'Whether to parse line breaks as &ltbr&gt, like GitHub does, without needing 2 spaces at the end of the line',
```

Comment:
Encodings should end with `;`
https://github.com/ivov/eslint-plugin-n8n-nodes-base/blob/89800fd5fcab91afdd955813dc055e3d79e4b543/lib/rules/node-param-description-unencoded-angle-brackets.ts#L18